drivers/stepper/ti/Kconfig

```
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 Navimatix GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+config STEPPER_TI
```

```suggestion
menuconfig STEPPER_TI
```
this will put all the TI drivers on the next page

---

This causes the compliance check to fail, as STEPPER_TI has no children. The first TI driver that adds corresponding KCONFIG entries should probably change it.